cmd/wallet-sdk-gomobile/docs/packages.md

```diff
@@ -0,0 +1,400 @@
+# Packages
```

Comment:
I recommend viewing this file directly on my branch instead of here to see the rendered view. Note that you can use the rich diff option above, but it shows a horizontal scrollbar that won't actually be there